nearcore/src/runtime/mod.rs

```diff
@@ -3199,4 +3201,27 @@ mod test {
         let view_state_value = view_state.get(&key).unwrap().unwrap();
         assert_eq!(state_value, view_state_value);
     }
+
+    /// Check that mainnet genesis hash still matches.
```

Comment:
Maybe expand this a bit? This assumes that the reader understands *why* we want that, might make sense to spent a sentence explaining that.